src/components/Header/UserSettings.ts

```diff
@@ -0,0 +1,188 @@
+import { css, html, LitElement } from "lit";
+import {
+  customElement,
+  property,
+  query,
+  queryAll,
+  state,
+} from "lit/decorators.js";
+import { localized, msg } from "@lit/localize";
+import { theme } from "../../utils/theme.ts";
+
+@customElement("bkd-user-settings")
+@localized()
+export class UserSettings extends LitElement {
+  @property()
+  currentLocale = "de";
+
+  @state()
+  protected open = false;
+
+  @query("button")
+  private menuButton: any;
+  @queryAll("a")
+  private menuLinks: any;
+
+  static styles = [
+    theme,
+    css`
+      :host {
+        display: flex;
+        position: relative;
+      }
+
+      button {
+        background: transparent;
+        border: none;
+        cursor: pointer;
+      }
+
+      ul {
+        position: absolute;
+        right: 0;
+        border: 1px solid var(--bkd-func-bg-grey);
+        list-style-type: none;
+        padding: 0.625rem 0;
+        margin-top: calc(32px + 0.5rem);
+        background: var(--bkd-func-bg-white);
+        z-index: 1;
```

Comment:
Da wär's noch cool die z-indices im `theme.ts` als CSS Property to haben, damit man eine Übersicht hat was es alles gibt. Ev. könnte man gleich einen allgemeinen `--bkd-z-index-dropdown` oder so einführen?